readme.md

```diff
@@ -1,57 +1,95 @@
 # RSpec Gladiator Arena
 
-The Emperor has commissioned you to build a Gladiator Arena. Your life depends on developing a well-thought-out arena, so you have been given tests to make sure
-you build it well.
+The Emperor has commissioned you to build a Gladiator Arena.
 
 ## How to do this assignment
 
-1. In the gladiator folder, run `bundle install` to make sure you have the right gems installed
-2. Run `rspec` to run your tests. They should all fail.
-3. Start with the `Gladiator` tests / class (see steps below)
-4. Then do the `Arena` tests / class (see steps below)
+The provided index.html file will be useful for testing out your code.
 
-For each individual test:
+## Part 1 - The Gladiator
 
-1. Read the error message.
-2. Make the smallest change in your code to fix the error.
-3. Run the test.
-4. Repeat 1 - 3 until the test passes
-5. Move to the next test.
+Create a `gladiator` class that has the following properties:
 
-## Bonus!
+* a `name`
+* a `weapon` (one of Spear, Club, Trident)
 
-Try writing the tests yourself:
+```js
```

Comment:
Creating a new `Gladiator` instance should look like:

or something about the end goal...